src/components/sideBarContentEdit/SelectField.tsx

```diff
@@ -222,7 +222,9 @@ export const OnlySelectField = ({
             gap={1}
             alignItems="center"
           >
-            <Text fontSize="14px">{speaker}</Text>
+            <Text textTransform={"capitalize"} fontSize="14px">
```

Comment:
Isn't this supposed to be lowercase instead of capitalize?

Reply:
@Emmanuel-Develops  suggested that we capitalize them to the user 

https://github.com/bitcointranscripts/transcription-review-front-end/pull/194#discussion_r1393826373

Reply:
I have no preference but we should stick with what the issue states. besides, its common to use lowercase tags than uppercase e.g. github
![image](https://github.com/bitcointranscripts/transcription-review-front-end/assets/60826700/d7647679-bc50-4ad6-84b6-91fb24c522dd)